spec/models/category_spec.rb

```diff
@@ -2,4 +2,61 @@
 
 describe Category do
   it { should have_many(:videos)}
+
+  describe "#recent_videos" do 
+    it "should return an empty arrayif there is no video" do
+      drama = Category.create(name: "Drama")
+      expect(drama.recent_videos).to eq([])
+    end
+
+    it "should return videos that belong in the same category" do
```

Comment:
The word 'should' in spec message is redundant - always use 3rd person of the verb. Read more about it in [this guide](http://betterspecs.org/#should)